app/controllers/application_controller.rb

```diff
@@ -6,7 +6,7 @@ class ApplicationController < ActionController::Base
   private
 
   def default_url_options
-    { locale: I18n.locale }
+    { locale: (I18n.locale == I18n.default_locale ? nil : I18n.locale) }
```

Comment:
Why locale for `uk` always is `nil`?

Reply:
This was added to avoid tests breaking. I was comparing URLs, but all comparisons failed due to "?locale=uk" added to the end of url. I've removed the default locale and now the specs started to work.
Here is a link to the StackOverflow thread https://stackoverflow.com/a/65918444
Please advise me if I can leave that as it is - or if there is some other approach :)

Reply:
It's bad path to fix code in order to tests passing. You can avoid problem similarly by adding parameter to path in tests: `admins_user_path(@user, locale: i18n.locale) `